docs/management/persistence.md

```diff
@@ -259,42 +259,35 @@ and starts appending new data into the new file.
 
 ### How I can switch to AOF, if I'm currently using dump.rdb snapshots?
 
-There is a different procedure to do this in version 2.0 and later versions, as you
-can guess it's simpler since Redis 2.2 and does not require a restart at all.
+If you want to enable AOF in a server that is currently using RDB snapshots,
+you need to convert the data interactively using `redis-cli`.
 
-**Redis >= 2.2**
+**IMPORTANT:** not following this procedure (e.g. just changing the config and
+restarting the server) can result in data loss!
+
+The following steps require at least Redis 2.2:
 
 * Make a backup of your latest dump.rdb file.
 * Transfer this backup to a safe place.
-* Issue the following two commands:
-* `redis-cli config set appendonly yes`
-* `redis-cli config set save ""`
-* Make sure your database contains the same number of keys it contained.
+* Enable AOF: `redis-cli config set appendonly yes`
+* Optionally disable RDB: `redis-cli config set save ""`
+* Make sure your database contains the same number of keys it contained
+  previously.
 * Make sure writes are appended to the append only file correctly.
+* Update your `redis.conf` and ensure that it matches the config above.
 
 The first CONFIG command enables the Append Only File persistence.
 
-The second CONFIG command is used to turn off snapshotting persistence. This is optional, if you wish you can take both the persistence methods enabled.
+The second CONFIG command is used to turn off snapshotting persistence. This is
+optional, if you wish you can take both the persistence methods enabled.
```

Comment:
We should avoid this change that just adding an extra line.

Reply:
I did that for consistency with the rest of the file, but I can revert the change.